disneyApi/src/components/Modal/ModalImg.jsx

```diff
@@ -0,0 +1,19 @@
+import styled from 'styled-components';
+
+const StyledModalImg = styled.img`
```

Comment:
This is the kind of structure I had in mind earlier. A file containing only styled components usually includes multiple components.